.rubocop.yml

```diff
@@ -8,6 +8,8 @@ AllCops:
 
 Style/Documentation:
   Enabled: false
+Style/HashSyntax:
+  EnforcedShorthandSyntax: "never"
```

Comment:
Rubocop [enforces this cop ](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax)since Ruby 3.1. 

```
# EnforcedShorthandSyntax 
{foo:, bar:}

# Not EnforcedShorthandSyntax 
{foo: foo, bar: bar}
```

I've turned it off because I think the latter is more readable to new Rubyists.